src/datatypes/character/Character.hpp

```diff
@@ -1,82 +1,154 @@
+
+            /**
+             * Setter for the health points of the character.
+             * @param hp Health points to set.
+             */
+            void setHp(unsigned int hp);
+
+            /**
+             * Getter for the intelligence points of the character.
+             * @return Current intelligence points of the character.
+             */
+            [[nodiscard]] unsigned int getIp() const;
+
+            /**
+             * Setter for the intelligence points of the character.
+             * @param ip Intelligence points to set.
+             */
+            void setIp(unsigned int ip);
+
+            /**
+             * Getter for the number of gambling chips owned by the character.
+             * @return Current number of gambling chips.
+             */
+            [[nodiscard]] unsigned int getChips() const;
+
+            /**
+             * Setter for the number of gambling chips owned by the character.
+             * @param chips Chip number to set.
+             */
+            void setChips(unsigned int chips);
+
+            /**
+             * Getter for the character properties.
+             * @return List of the character properties.
+             */
+            [[nodiscard]] const std::list<PropertyEnum> &getProperties() const;
+
+            /**
+             * Setter for the character properties.
+             * @param properties List with the character properties that should be set.
+             */
+            void setProperties(const std::list<PropertyEnum> &properties);
+
+            /**
+             * Getter for the gadgets owned by the character.
+             * @return List with the current gadgets of the character.
+             */
+            [[nodiscard]] const std::list<gadget::Gadget> &getGadgets() const;
+
+            /**
+             * Setter for the gadgets of the character.
+             * @param gadgets List of gadgets to set.
+             */
+            void setGadgets(const std::list<gadget::Gadget> &gadgets);
+
+        private:
+            const spy::util::UUID characterId;
+            const std::string name;                                     ///< Character name.
+            spy::util::Point coordinates;                               ///< Current coordinate of the character.
+            unsigned int mp;                                            ///< MovePoints.
+            unsigned int ap;                                            ///< ActionsPoints.
+            unsigned int hp;                                            ///< HealthPoints.
+            unsigned int ip;                                            ///< IntelligencePoints.
+            unsigned int chips;                                         ///< Number of gambling chips.
+            std::list<PropertyEnum> properties;                         ///< Character properties.
+            std::list<spy::gadget::Gadget> gadgets;                     ///< Gadgets owned by the character.
```

Comment:
* Was macht `///<`?

* Ich bin gegen diese Art von Kommentaren (in gleicher Zeile und aligned), da das kaputtgeht sobald man ein Zeichen ändert... Außerdem ist der Text maximal weit vom code entfernt.

* Vorschlag (Um Diskussion zu vermeiden): Bytes kosten nix, warum machen wir hier nicht einfach die Namen der Variablen aussagekräftiger (also statt `mp` lieber `movePoints` usw.)? Dann spart man sich die Kommentare, und es wird überall lesbarer.

Reply:
Zu dem ///<: das ist so notwendig um einen Doxygen Kommentar in dieselbe Zeile zu bringen (siehe [hier](https://stackoverflow.com/questions/4428038/doxygen-comments-at-end-of-line))

Den Vorschlag finde ich gut, dann kann man das ExtractPrivate bei Doxygen auch wieder raus nehmen. Bei den anderen habe ich die Kommentare lediglich ergänzt, damit es konsistent ist. Aber du hast schon recht, wirklich notwendig sind die da nicht.